packages/falcon-sensor/falcon-sensor.patch

```diff
@@ -0,0 +1,10 @@
+diff -x '*.tgz' -x '*.lock' -uNr packages/falcon-sensor/charts-original/Chart.yaml packages/falcon-sensor/charts/Chart.yaml
+--- packages/falcon-sensor/charts-original/Chart.yaml
++++ packages/falcon-sensor/charts/Chart.yaml
+@@ -20,3 +20,6 @@
+ - https://github.com/CrowdStrike/falcon-helm
+ type: application
+ version: 0.8.1
++annotations:
++  catalog.cattle.io/certified: partner
++  catalog.cattle.io/release-name: falcon-helm
```

Comment:
Was this intended to be committed?

Reply:
Yes. This is intentional. See https://github.com/rancher/partner-charts/blob/main-source/packages/sysdig/sysdig.patch

Reply:
Ah. main-source vs main. Thanks!